docs/visual-basic/programming-guide/concepts/linq/how-to-split-a-file-into-many-files-by-using-groups-linq.md

```diff
@@ -62,7 +62,7 @@ Class SplitWithGroups
         ' Note that nested foreach loops are required to access  
```

Comment:
It would be useful to add an `Includes System.Linq` above line 44 rather than mention it in the "Compiling the Code section".